File: examples/swagger-3/src/test/resources/asserts/typescript/umi_request/src/resp/ExampleDTO.ts
/* tslint:disable */
/* eslint-disable */

        import {TestMethodDTO} from "./TestMethodDTO";
import {DefaultOrderField} from "feign-client";



export interface  ExampleDTO {

            /**
                *字段在java中的类型为：Integer
            **/
        querySize?: number;
            /**
                *字段在java中的类型为：Integer
            **/
        queryPage?: number;
            /**
                *字段在java中的类型为：String
            **/
        name?: string;
            /**
                *字段在java中的类型为：String
            **/
        keyword?: string;
            /**
                *字段在java中的类型为：TestMethodDTO
            **/
        method?: TestMethodDTO;
}

File: examples/swagger-3/src/test/resources/asserts/java/retrofit/src/com/wuxp/codegen/swagger3/resp/ExampleDTO.java
package com.wuxp.codegen.swagger3.resp;

import lombok.Data;
import lombok.experimental.Accessors;
import jakarta.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;
        import com.wuxp.codegen.swagger3.resp.TestMethodDTO;

@Data
@Accessors(chain = true)
public class  ExampleDTO {

            /**
                *字段在java中的类型为：Integer
            **/
         Integer querySize;

            /**
                *字段在java中的类型为：Integer
            **/
         Integer queryPage;

            /**
                *字段在java中的类型为：String
            **/
        private String name;

            /**
                *字段在java中的类型为：String
            **/
        private String keyword;

            /**
                *字段在java中的类型为：TestMethodDTO
            **/
        private TestMethodDTO method;

}
